### ISSUE TITLE
Quest tracked isn't updating.

### ISSUE THREAD

**Issue by Akirabane** [Wynntils/Wynntils#2214]
When you track a quest, after talking to an NPC, and having a new objective to the quest, it won't update automatically. You have to open the content book, go in quests, un-track it, then track it back to update the beacon showing the next step.

**Comment by kristofbolyai** [comment]
Artemis version?

**Comment by Akirabane** [comment]
Fabric -> v0.0.4 beta.48

**Comment by Akirabane** [comment]
Happens motsly on starting a quest, it won't update the first time. Then, some times it updates, sometimes not

**Comment by kristofbolyai** [comment]
Well we read quest info from the scoreboard, not the quest book.

**Comment by Akirabane** [comment]
Yes, so I tested to see if it was a bug from Wynncraft, without Wynntils mod, and the quest on scoreboard is updating automatically. Came back using wynntils, and the problem was still here, I started a random quest and... not updating, still have to un-track the quest, then track it back.

**Comment by JoonaFinland** [comment]
I can vouch that this happens for me too. The actual Wynncraft content book updates the quest steps correctly, but in Wynntils I need to retrack once-ish per quest.

**Comment by DcAttekum** [comment]
I was looking to see if anyone else was having this problem. Can confirm that this is indeed an issue. Using wynntils 0.0.4-beta 82 fabric.

**Comment by koromaki88** [comment]
Encountering this problem again now. I have to untrack and retrack to update quest tracking. Unsure how to consistently reproduce the issue since it happens periodically. Cannot reproduce it in vanilla Minecraft with scoreboard, so I don't think it's on Wynncraft's end.

**Version**: Wynntils (fabric) v2.0.7, Fabric Loader 0.16.0, Minecraft 1.21.1 - Tested with Wynntils only and no other mods.

![2024-08-18_03 31 42](https://github.com/user-attachments/assets/44c1833e-1440-4b8b-936f-2f88f0c9148f)
![2024-08-18_03 32 08](https://github.com/user-attachments/assets/b219f084-ffc7-4ca2-9988-b17c8044edea)
![2024-08-18_03 58 52](https://github.com/user-attachments/assets/ec894b41-8eee-41b1-9592-5ca82537a5c0)
![2024-08-18_03 59 13](https://github.com/user-attachments/assets/ae2f765c-5f44-4644-882c-3300ee887c27)

**Comment by kristofbolyai** [comment]
It's an older Wynn bug that happens when you track a quest before starting it.

**Comment by ShadowCat117** [comment]
Actually, I discovered recently it's caused by a toggle. If you run `/toggle autotracking` you should get a message saying "Your content will now auto-track." and it should work from now on.

**Comment by koromaki88** [comment]
> Actually, I discovered recently it's caused by a toggle. If you run `/toggle autotracking` you should get a message saying "Your content will now auto-track." and it should work from now on.

Thanks. This seems to fix the issue.

**Comment by magicus** [comment]
Maybe we should send that command from the mod? Or at least warn if it is off? (I realize this probably means sending the command twice, just to read the current status, unless Wynn has added a way to read your toggle settings lately.)

**Comment by ShadowCat117** [comment]
Not a Wynntils issue https://github.com/Wynntils/Wynntils/issues/2214#issuecomment-2294995863
